spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/admin/SpannerSchemaUtils.java

```diff
@@ -47,76 +52,47 @@
 
 	private final SpannerTypeMapper spannerTypeMapper;
 
+	private final boolean createTableDdlDeleteOnCascade;
```

Comment:
This is still different from `createTableStatementsCascadeOnDelete`. Also, I think a better name would actually indicate that this is for interleaved tables. Maybe `createInterleavedTableCascadeOnDelete`?

Reply:
Yes I can change to that new name everywhere. (originally I felt it was OK to have "Ddl" in this class because so many things in this class have those letters in the name, and it would have more meaning given the context of this class and all the "Ddl" related functions.)

Reply:
`createTableStatementsCascadeOnDelete` to use `DDL` in the name.
Actually, I'm leaning towards this name now:
`createInterleavedTableDdlOnDeleteCascade`

Reply:
Renamed in all 3 places to `createInterleavedTableDdlOnDeleteCascade` and merged in other PRs and resolved conflicts. 

I'll update the ref doc PR for this new name, too.